resources/building-mobile-extensions/event-processing.md

```diff
@@ -8,7 +8,9 @@ An event is composed of the following information:
 * Event data, which is the structured data that is associated with this particular event.   This data is retrievable as a JSON blob.
 * Versioning information, which is used to synchronize a group of events with the associated shared data, is not directly exposed to extensions.  This information is implicitly used when getting shared states.
 
-**Tip**: Extensions use the `ACPExtensionEvent` \(iOS\) / `Event` \(Android\) class to create events and access the information the modules hold.
+**Tip**: Extensions use the ACPExtensionEvent (iOS) / Event (Android) class to create events to dispatch to the extensions in order to make requests or retrieve data.
```

Comment:
Extensions use the `ACPExtensionEvent` (iOS) / `Event` (Android) class to create events and dispatch them to the Mobile SDK event hub in order to trigger the extension business logic or to retrieve data.

Reply:
Please remove "in order" and reword as follows:

"Extensions use the ACPExtensionEvent (iOS) / Event (Android) class to create events and dispatch them to the Mobile SDK event hub to trigger the extension business logic or to retrieve data."